tools/ea/Ea.hs

```diff
@@ -170,38 +170,39 @@ genPayloadModule' v tag cwTxs =
                     execNewGenesisBlock noMiner (V.fromList cwTxs)
 
             let payloadYaml = TE.decodeUtf8 $ Yaml.encode payloadWO
-                modl = T.unlines $ startModule tag <> [payloadYaml] <> endModule
```

Comment:
why this change?

Reply:
This is the change that causes the generated payload modules to not depend on neat-interpolation. The code replaces the use of the QuasiQuote with `T.unlines`, `T.intercalate`, and `<>` for `Text`, which are expected to be stable.